django_pgviews/view.py

```diff
@@ -64,8 +64,104 @@ def realize_deferred_projections(sender, *args, **kwargs):
 models.signals.class_prepared.connect(realize_deferred_projections)
 
 
+def _schema_and_name(connection, view_name):
+    if "." in view_name:
+        return view_name.split(".", 1)
+    else:
+        try:
+            schema_name = connection.schema_name
+        except AttributeError:
+            schema_name = "public"
+
+        return schema_name, view_name
+
+
+def _create_mat_view(cursor, view_name, query, params, with_data):
+    """
+    Creates a materialized view using a specific cursor, name and definition.
+    """
+    cursor.execute(
+        "CREATE MATERIALIZED VIEW {0} AS {1} {2};".format(
```

Comment:
Do we have to worry about SQL injection here? I know its kind of a deal when parsing strings like this, but not super clued up on it.

Reply:
I think you kind of have to do it like that here, because if you used the arguments the entire sql would be a single escaped string here, which would then not read as a query. 

Since this is entirely-code defined (and not user defined) this is ok.

Reply:
Yeah, thought as such. Think that is okay :)